Mathlib/CategoryTheory/Galois/Examples.lean

```diff
@@ -0,0 +1,106 @@
+/-
+Copyright (c) 2024 Christian Merten. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christian Merten
+-/
+import Mathlib.CategoryTheory.Galois.Basic
+import Mathlib.RepresentationTheory.Action.Basic
+import Mathlib.RepresentationTheory.Action.Limits
+import Mathlib.CategoryTheory.Limits.FintypeCat
+import Mathlib.CategoryTheory.Limits.Shapes.Types
+import Mathlib.Logic.Equiv.TransferInstance
+
+/-!
+# Examples of Galois categories and fibre functors
+
+We show that for a group `G` the category of finite `G`-sets is a `PreGaloisCategory` and that the
+forgetful functor to `FintypeCat` is a `FibreFunctor`.
+
+## Todo
+
+* Characterize connected objects in the category of finite `G`-sets as those with transitive
+  `G`-action
+
+-/
+
+universe u v w
+
+namespace CategoryTheory
+
+namespace FintypeCat
+
+open Limits Functor PreGaloisCategory
+
+private noncomputable def imageComplement {X Y : FintypeCat.{u}} (f : X ⟶ Y) :
+    FintypeCat.{u} := by
+  haveI : Fintype (↑(Set.range f)ᶜ) := Fintype.ofFinite _
+  exact FintypeCat.of (↑(Set.range f)ᶜ)
+
+private def imageComplementIncl {X Y : FintypeCat.{u}}
```

Comment:
I am not sure the definitions in this file need to be private.

Reply:
I removed the `private` tag.